forem/forem
# ✂️ ✂️  Removed colour picker from onboarding as it's never used ✂️ ✂️

## What type of PR is this? (check all applicable)


## Description

The colour picker is not currently used in the onboarding flow.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

The onboarding flow should still work

1. Log in as anyone
2. Navigate to http://localhost:3000/onboarding
3. Go through the flow.
4. No errors should occur and the onboarding should complete successfully.

### UI accessibility concerns?

N/A removing dead code 

## Added/updated tests?

- [x] No, and this is why: N/A removing dead code and there are existing specs

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/42D3CxaINsAFemFuId/giphy.gif)
